config/routes.rb

```diff
@@ -6,6 +6,8 @@
 
   get "map", to: "map#show"
 
+  get "bulletins", to: "bulletins#show"
```

Comment:
I wonder if we ought to be clear about whether we are showing 3 bulletins (one for each day) or 1 bulletin which covers 3 days. If the former (3 bulletins on one page) then I think we're showing a list:

`get :bulletins, to: "bulletins#index"`

But if it's a single bulletin describing 3 days then it's a singular resource:

`get :bulletin, to: "bulletins#show"`

It think either makes sense but that we ought use "show" for a single resource and "index" for a list.

Reply:
mmmm good point. My instinct is it's a single bulletin describing 3 days, so I'll go for the latter 👍

Reply:
but then again, the reason I went for "bulletins" isn't because each 3 day forecast represents multiple bulletins, it was because the 3 day forecast for each area represents a distinct bulletin...